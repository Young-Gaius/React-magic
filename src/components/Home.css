
.home-container {
  height: 100vh;
  display: flex;
  flex-direction: column;
  margin-left: 16px;
}

.home-title {
  font-size: 2rem;
  font-weight: bold;
  margin-top: 2rem;
}

.home-description {
  font-size: 1.25rem;
  margin-top: 1rem;
  width: 100%;
}

.home-subtitle {
  font-size: 1rem;
  margin-top: 1rem;
}
