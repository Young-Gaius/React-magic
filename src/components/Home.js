import React from 'react';
import './Home.css'; // Import the CSS file

function Home() {
  return (
    <div className="home-container">
      <h1 className="home-title">Welcome to Math Magicians</h1>
      <p className="home-description">
        Math Magicians is a web application that provides a calculator for performing arithmetic
        calculations and a random quote generator to inspire and motivate you.
      </p>
      <p className="home-subtitle">
        Get started by exploring the calculator functionality or discovering insightful quotes.
      </p>
    </div>
  );
}

export default Home;
