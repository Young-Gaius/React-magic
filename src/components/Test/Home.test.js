import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../Home';

describe('Home', () => {
  test('renders correctly', () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});