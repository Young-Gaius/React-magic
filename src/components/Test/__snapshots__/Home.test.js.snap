// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<div
  className="home-container"
>
  <h1
    className="home-title"
  >
    Welcome to Math Magicians
  </h1>
  <p
    className="home-description"
  >
    Math Magicians is a web application that provides a calculator for performing arithmetic calculations and a random quote generator to inspire and motivate you.
  </p>
  <p
    className="home-subtitle"
  >
    Get started by exploring the calculator functionality or discovering insightful quotes.
  </p>
</div>
`;
